int32 position_1
int32 position_2
int32 position_3
int32 position_4
int32 position_5
int32 position_6
---
bool success
